src/disclaimer/Disclaimer.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+
+const SMALL_DATA = (
+  <>
+    * Please always take note the number of people associated with each
```

Comment:
"Please always take note of the number..." ?

Reply:
good catch